### PR TITLE
Add meta files

### PR DESCRIPTION
**Pull request by konoikon**
Modified the files according to comments

### CODE REVIEW

**Comment by pganssle** [comment]
@konoikon Your history here is still wrong in some way, but now that it's a new branch we can fix it. I can help you work through the git "surgery" if you want.

Also, you will want to add `.DS_STORE` to your global ignore patterns [see here](https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore).

File: AUTHORS.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+This file contains a list with all the contributors to the *grid strategy* project.
+
+Contributors
+------------
+
+- Paul Ganssle <paul@ganssle.io>
+- Konstantinos Oikonomou <kons.oikonomou@gmail.com>
```
Comment (pganssle) [inline AUTHORS.md]: @Gurnek How should we list you in the AUTHORS file? What, if any, e-mail do you want?

Comment (Gurnek) [reply AUTHORS.md]: @pganssle you can just write the following
Gurnek Singh Mokha <gurnekmokha96@gmail.com>

Comment (pganssle) [reply AUTHORS.md]: @konoikon Can you update the AUTHORS file in this PR accordingly?

**Pull request merged**